.github/workflows/prepare_release.yml

```diff
@@ -11,28 +11,28 @@ jobs:
     name: Tag release
     runs-on: ubuntu-latest
     outputs:
-      is_release: ${{ steps.check_number_of_commits.outputs.is_release }}
+      is_release: ${{ steps.check-number-of-tags.outputs.is_release }}
     steps:
       - name: Check out project sources
         uses: actions/checkout@v3
         with:
           fetch-depth: 2
       - name: Set up Git user
-        uses: fregante/setup-git-user@v1
+        uses: fregante/setup-git-user@v2
       - name: Tag new versions of Pulumi Kotlin (if available)
         run: ./gradlew tagRecentRelease
-      - name: Check number of commits
-        id: check_number_of_commits
+      - name: Check number of tags
+        id: check-number-of-tags
         run: |
-          number_of_commits=$(git tag --points-at HEAD | wc -l | awk '{print $1}')
-          if [[ "$number_of_commits" == "0" ]]; then is_release=false; else is_release=true; fi
+          $number_of_tags=$(git tag --points-at HEAD | wc -l | awk '{print $1}')
+          if [[ "$number_of_tags" == "0" ]]; then is_release=false; else is_release=true; fi
```

Comment:
This is not really about this PR, but I'm not sure if I understand this piece. Maybe we could improve the name by expressing the intent?